docs/webdriver-bidi.md

```diff
@@ -152,8 +161,6 @@ This is an exciting step towards a more unified and efficient cross-browser auto
   - Input.dragOver()
   - Input.drop()
   - Page.authenticate()
```

Comment:
Still here as I want to work on fixing the issue for Firefox as they are on our side.